packages/bierzo-wallet/src/routes/welcome/index.tsx

```diff
@@ -33,8 +35,15 @@ const onPayment = (): void => {
 
 const Welcome = (): JSX.Element => {
   const toast = React.useContext(ToastContext);
+  //TODO: Fix this as soon as proper react-redux definitions will be available
+  const dispatch = (ReactRedux as any).useDispatch();
   const classes = useStyles();
 
+  const onConnect = async (): Promise<void> => {
+    const request = generateGetIdentitiesRequest();
+    dispatch(getExtension(request));
```

Comment:
instead of `getExtension`, I would say: `setExtensionStatus`.

Reply:
If we want to remove Thunk then we do not need this action and call.

Reply:
But what you need to do here is update the redux store with the received information. Once you create the login view which is one of the ticket with the biggest priority (because it unblocks many things), you will remove this from here, but for now, the place where we "log in" is here, right?

Reply:
Yes, you are right